.pre-commit-config.yaml

```diff
@@ -0,0 +1,63 @@
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v1.11.0
+    hooks:
+      - id: helm-docs-built
+        name: 'Helm Docs for Turing Chart'
+        files: '^infra/charts/turing/'
+        args:
+          - --chart-search-root=infra/charts/turing
+          - --template-files=./README.md.gotmpl
+      - id: helm-docs-built
```

Comment:
Is it expected that multiple hooks have the same `id`?

Reply:
Yes, the `id` is simply used to match the hook from the published repo, as there can be multiple. Eg: https://github.com/norwoodj/helm-docs/blob/master/.pre-commit-hooks.yaml

To 'distinguish' one hook from another where the same id is used, the `alias` property can be set (which I've now added to link to the respective Go modules' `make lint`:

```yaml
     - id: golangci-lint
        name: 'Linter for API'
        alias: golangci-lint-api
        files: '^api/'
        types: [go]
        entry: bash -c 'cd api && golangci-lint run --timeout 3m --fix'
      - id: golangci-lint
        name: 'Linter for the Router Engine'
        alias: golangci-lint-router
        files: '^engines/router/'
        types: [go]
        entry: bash -c 'cd engines/router && golangci-lint run --fix'
```